### DISCUSSION TITLE
Setting `expandtab` to true doesn't affect `:set expandtab?` / How to check what file last assigned to a neovim option?

### DISCUSSION THREAD

**Discussion by starptr** [nix-community/kickstart-nix.nvim#44] [Q&A] [answered]
In the template `init.lua`, one of the lines sets `expandtab`:
```lua
opt.expandtab = true
```
but when I run `:set expandtab?`, it returns `noexpandtab`. What could possibly explain this?

**Comment by mrcjkb** [comment, marked as answer]
Running this without any modifications:

```
:verbose set expandtab?
  expandtab
        Last set from Lua (run Nvim with -V1 for more details)
```

There are some options that will set `noexpandtab`, so I suggest you use `:verbose set expandtab?` to troubleshoot that.

**Reply by starptr** [reply]
Thanks for the help! This told me that in my scenario the built-in editorconfig handler was setting it.

For posterity: anyone can repro this and see this behavior:
```lua
nix run "github:nix-community/kickstart-nix.nvim" -- -V1 nvim/init.lua
:verbose set expandtab?
```
or really any lua file such that none of its parent directories contain an editorconfig file. I think neovim's built-in editorconfig plugin will default to setting `noexpandtab` when it doesn't find an editorconfig file. This sounds like a bug to me but I'm not sure yet.

**Reply by mrcjkb** [reply]
Are you sure? 
There was no .editorconfig file where I tested it.
